fe/fe-core/src/test/java/com/starrocks/sql/plan/DecimalTypeTest.java

```diff
@@ -212,4 +211,27 @@ public void testArrayAggDecimal() throws Exception {
             connectContext.getSessionVariable().setNewPlanerAggStage(stage);
         }
     }
+
+    @Test
+    public void testDecimalV2ArraySlice() throws Exception {
```

Comment:
Why not use SQL-test to covert this case?

Reply:
already have case